naga/src/back/wgsl/writer.rs

```diff
@@ -58,6 +59,8 @@ bitflags::bitflags! {
     pub struct WriterFlags: u32 {
         /// Always annotate the type information instead of inferring.
         const EXPLICIT_TYPES = 0x1;
+        /// Insert polyfills for matrix inverse functions.
+        const POLYFILL_INVERSE = 0x2;
```

Comment:
I don't think we need this new flag, if it's not enabled and the function is used we will error. I don't see a situation where that would be more useful than just injecting the polyfill (considering the SPIR-V/GLSL -> WGSL use case).

cc @jimblandy for thoughts

Reply:
Yeah, if we're going to generate code that uses `Inverse`, then we might as well just supply it.

Reply:
Removed as of b698a8 to unconditionally generate the polyfill